app/src/components/Box.css

```diff
@@ -0,0 +1,40 @@
+.box {
+  padding: 0.5rem;
+  width: 25rem;
+  height: 475px;
+  margin: 5rem auto;
+  background: #1F2630;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 30px;
+  text-align: center;
+}
+
+.box h3 {
+  font-family: 'Arial';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 30px;
+  line-height: 41px;
+}
+
+.box p {
+  font-family: 'Arial';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  padding: 0 2rem;
+  line-height: 25px;
+  letter-spacing: 0.02em;
+  color: #98A1B1;
+}
+
+@media (max-width: 475px) {
+  .box {
+    width: 20rem;
+    height: 490px;
```

Comment:
its a media query....maybe u can get away with pixel...but ideally use rem, esp when u have width using rem already....stick to 1